features/bevragen/persoon-beperkt/verblijfplaats/dev/autorisatie-gba.feature

```diff
@@ -0,0 +1,181 @@
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam       | waarde                      |
+      | type       | ZoekMetPostcodeEnHuisnummer |
+      | postcode   | 2497BV                      |
+      | huisnummer | 103                         |
+      | fields     | verblijfplaats.functieAdres |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | detail   | De foutieve fields waarden zijn: fields[0].                             |
+      | code     | authorization                                                           |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+      | ad hoc rubrieken                          | missende autorisatie        |
+      | 10120 81010 81030 81120 81160 81210 81310 | straat (81110)              |
+      | 10120 81010 81030 81110 81120 81160 81310 | locatiebeschrijving (81210) |
+      | 10120 81010 81030 81110 81120 81160 81210 | land (81310)                |
+   
+  
+  Rule: de 'verblijfplaatsBinnenland' field alias moet worden gebruikt door een consumer die niet is geautoriseerd voor het bevragen van 'verblijfplaats buitenland' velden
+
+    Scenario: Afnemer vraagt om alleen binnengemeentelijke verblijfplaatsgegevens en is niet geautoriseerd voor verblijf buitenland
+      Gegeven de afnemer met indicatie '12345' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60)                                   | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | 10120 80910 81110 81115 81120 81130 81140 81150 81160 81170 81210 | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 12345  |
+      | gemeenteCode | 0518   |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                       |
+      | type                | RaadpleegMetBurgerservicenummer              |
+      | burgerservicenummer | 000000024                                    |
+      | fields              | burgerservicenummer,verblijfplaatsBinnenland |
+      Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
```

Comment:
Ook hier is de lijst van geleverde verblijfplaats properties niet compleet.

Reply:
wel na wijzigen naar ZoekMetPostcodeEnHuisnummer, dus PersoonBeperkt